tests/network/dual_stack_cluster.go

```diff
@@ -9,7 +9,7 @@ import (
 	"kubevirt.io/kubevirt/tests/libnet"
 )
 
-var _ = SIGDescribe("Dual stack cluster network configuration", func() {
+var _ = SIGDescribe("[Sig-Network]Dual stack cluster network configuration", func() {
```

Comment:
This and other modules are already tagged through SIGDescribe.

Reply:
done